slides/week3/git.org

```diff
@@ -25,12 +28,26 @@
 - Branches are what people have the most trouble with
   - _*Ask questions /please/!*_ This is a difficult topic.
 
-** Git As a Graph
+** Remotes vs Branches vs Forks
+- A fork or clone does a full copy of the git repo
+- A remote is a pointer to another copy of this git repo
+- A repository is a collection of *branches*
```

Comment:
This is misleading - it implies that branches are fundamental to the internal structure to git, while in reality they are much more just pointers to the underlying structure. 

For example, you can use a git entirely without branches, if you wished, but the repository would still exist.